src/particle_spinors.jl

```diff
@@ -3,6 +3,7 @@ import Base.getindex
 
 const SPINOR_VALIDITY_CHECK = Ref(true)
 
+
```

Comment:
Please run the formatter script. This removes the extra line.

Reply:
Done in 467cfee.